styles.css

```diff
@@ -0,0 +1,698 @@
+  color: #fff;
+  font-size: 20px;
+  margin: 2rem;
+  align-items: center;
+  text-decoration: underline;
+}
+
+.program-cards {
+  padding: 20px;
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.program-cards .card_item {
+  padding: 1px;
+  width: 15%;
+  background-color: rgba(255, 255, 255, 0.1);
+  justify-content: center;
+  margin: 2px;
+  border: solid transparent 3px;
+  transition-duration: 400ms;
+}
+
+.program-cards .card_item:hover {
+  border: solid white 3px;
+}
+
+.program-cards .card_inner {
+  border-radius: 5px;
+  min-width: 100%;
+  height: auto;
+  text-align: center;
+}
+
+.program-cards .card_item img {
+  color: var(--orange);
+  width: 80px;
+  height: 80px;
+  margin-bottom: 5px;
+}
+
+.program-cards .card_item .card_title {
+  color: var(--orange);
+  font-family: var(--font-lato);
+  font-weight: 700;
+  font-size: 18px;
+}
+
+.program-cards .card_item .card-text {
+  font-size: 14px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.footer .footer1 {
+  background-color: #3d3d3d;
+  align-items: center;
+  height: auto;
+  text-align: center;
+  width: 100%;
+  padding: 1rem;
+  overflow-x: hidden;
+}
+
+.footer hr {
+  border: solid var(--orange) 1px;
+  background: #ccc;
+  align-items: center;
+  width: 3%;
+  margin: auto;
+}
+
+.footer .footer1 ul img {
+  width: 120px;
+  height: 30px;
+  margin-left: 4rem;
+}
+
+.footer .footer1 ul {
+  margin-top: 4rem;
+}
+
+.footer-header hr {
+  color: var(--orange);
+}
+
+.footer2 {
+  padding: 10px;
+  font-family: var(--font-lato);
+  color: var(--dark);
+}
+
+.footer2 .footer-logo {
+  margin-right: 14rem;
+  margin-left: 14rem;
+}
+
+.footer-description {
+  margin: 1rem 0;
+}
+
+/* Speakers Section */
+
+.speakers {
+  background-color: #f9f9f9;
+  text-align: center;
+  font-family: var(--font-lato);
+  padding: 20px;
+}
+
+.speaker-heading h5 {
+  font-size: 20px;
+  color: var(--dark);
+  margin-bottom: 1rem;
+}
+
+.presidentsDiv hr {
+  border: solid #ccc 0.8px;
+  width: 13%;
+  margin-top: 0.5rem;
+}
+
+.speakers .speaker-heading hr {
+  border: solid var(--orange) 1px;
+  align-items: center;
+  width: 3%;
+  margin: auto;
+}
+
+.presidentsDiv {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  margin: auto;
+  width: 80%;
+  overflow-x: hidden;
+}
+
+.presidentsDiv .imageContainer {
+  display: flex;
+  width: 100%;
+  margin: 1rem;
+}
+
+.presidentsDiv .imageContainer .imagedots {
+  width: 50px;
+}
+
+.presidentsDiv .imageContainer .presidentImage {
+  width: 150px;
+  height: 100px;
+  margin-right: 1rem;
+}
+
+.presidentDetails {
+  text-align: left;
+  color: var(--orange);
+  font-style: italic;
+  font-weight: 800;
+  font-size: 12px;
+}
+
+.infoText {
+  width: 250px;
+}
+
+.presidentName {
+  text-align: left;
+  margin-bottom: 0.5rem;
+}
+
+.infoContainer {
+  margin-left: 1rem;
+}
+
+#show-more {
+  display: none;
+}
+
+/* Mobile styling begins here */
+
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] Kindly make your page mobile fast  that is the mobile design fast the using media queries to make the desktop design, currently its the other way around